OpenRA.Mods.Common/Lint/CheckPlayers.cs

```diff
@@ -51,6 +51,14 @@ public void Run(Action<string> emitError, Action<string> emitWarning, Map map)
 				if (map.SpawnPoints.Value.Distinct().Count() != spawnCount)
 					emitError("Duplicate spawn point locations detected.");
 			}
+
+			foreach (var kv in map.ActorDefinitions)
+			{
+				var actorReference = new ActorReference(kv.Value.Value, kv.Value.ToDictionary());
+				var ownerName = actorReference.InitDict.Get<OwnerInit>().PlayerName;
+				if (!playerNames.Contains(ownerName))
+					emitWarning("Actor {0} is owned by unknown player {1}.".F(actorReference.Type, ownerName));
```

Comment:
Already mentioned in the chat, but for completeness' sake: please turn this into an `emitError`, so that Travis will properly flag the build as failed.